_includes/posts-by-date.html

```diff
@@ -1,8 +1,8 @@
 <section class="posts-by-date">
     <h2>Recent Posts</h2>
-    <ol>
+    <ul class="post-list">
```

Comment:
It doesn't matter, but i think this should still be an "ol", since it's ordered by date 😛